sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosPermissionProperties.java

```diff
@@ -6,7 +6,9 @@
 import com.azure.cosmos.implementation.Constants;
 import org.apache.commons.lang3.StringUtils;
 
+import java.nio.charset.StandardCharsets;
```

Comment:
I believe this is an unused import, if so, please get rid of it :)

Reply:
removed in the lastest [commit](https://github.com/Azure/azure-sdk-for-java/pull/9111/commits/b972e8c088bb80842e7fa0633740f49128f1991b).